test/files/run/issue-11614.scala

```diff
@@ -0,0 +1,3 @@
+object Test extends App {
```

Comment:
This should not be a `run` test. I would augment existing test in test/files/neg/t6276.scala